File: Treinamento/C/Estrutura_de_repetição/Questão01.c
#include <stdio.h>

int main() {
    int n;

    printf("Digite a nota desejada: ");
    scanf("%i", &n);

    while (n < 0 || n > 10) {
        printf("Número digitado inválido, digite novamente\n");
        printf("Digite a nota desejada: ");
        scanf("%i", &n);
    }
    printf("Número digitado: %i\n", n);
}

File: Treinamento/Java/Estrutura_de_repetição/Questão01.java
import java.util.Scanner;
public class Questão01 {
    public static void main(String[] args) {
        Scanner s;
        s = new Scanner(System.in);

        System.out.print("Digite a nota: ");
        double nota = s.nextDouble();

        while(nota < 0 || nota > 10) {
            System.out.println("Nota inválida, digite novamente");
            System.out.print("Digite a nota: ");
            nota = s.nextDouble();
        }

        System.out.println("Nota digitada: " + nota);
    }
}
   
